setup.cfg

```diff
@@ -0,0 +1,21 @@
+[metadata]
+name = ovn-kubernetes
+summary = OVN integration for Kubernetes
+description-file = README.md
+author = OVN
+author-email = dev@openvswitch.org
+classifier =
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: Apache Software License
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+
+[files]
+packages = ovn_k8s
```

Comment:
> should this be ovn-kubernetes instead of ovn-k8s?

I am not sure what the meaning of "packages" here means? If this is the python module, then ovn_k8s is right. I will ignore this for the time being, till I understand this better.